.github/workflows/hatch_ci.yml

```diff
@@ -13,13 +13,38 @@ on:
         required: false
         default: "3.12"
         description: "Maximum Python version to test against."
+      whitesource_product_name:
+        type: string
```

Comment:
Changing inputs for this workflow will currently fail ci for:
`solace-ai-connector|-slack|rest|web`
`cognitive-mesh` repos. 
Is it possible to set a default value for these based on the repo name ? 
For example  with `{{github.repository}}`

Reply:
Since its required false. it shouldn't break any existing ci. I tested it here: https://github.com/SolaceDev/solace-ai-connector-rest/actions/runs/12125384637/job/33805226594

Running whitesource is controlled by the presence of `whitesource_product_name` and `whitesource_project_name`.  Those will be required parameters to upload the results.